stdlib/public/core/Indices.swift

```diff
@@ -81,6 +81,23 @@ extension DefaultIndices: Collection {
   public var indices: Indices {
     return self
   }
+  
+  @inlinable
+  public func index(_ i: Index, offsetBy distance: Int) -> Index {
+    return _elements.index(i, offsetBy: distance)
+  }
+
+  @inlinable
+  public func index(
+      _ i: Index, offsetBy distance: Int, limitedBy limit: Index
```

Comment:
Formatting nit:

```suggestion
    _ i: Index, offsetBy distance: Int, limitedBy limit: Index
```

(See line 28)